app/models/transaction.rb

```diff
@@ -112,6 +117,10 @@ def find_customer_name_by_id(customer_id)
     ''
   end
 
+  def transfer?
+    category_id == Category.receipt_id && category_id.present?
```

Comment:
@yunixon You may skip condition `category_id.present?` because if `category_id` is nil it will not equal `Category.receipt_id`